src/platforms/unity/configuration/native-support.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Native Support
+description: "Learn how the Unity SDK handles native support."
+sidebar_order: 30
+---
+
+<Note>
+
+The native support requires configuration using the Sentry Unity Editor window. Support for programatic initialization via C# is planned.
+
+</Note>
+
+<Note>
+
+Our Unity SDK currently supports native iOS when building for device only. Support for the iOS simulators is planned.
```

Comment:
This note might be worth adding further down. The goal is to reduce these notes from the top of the docs

Or/and throwing the error message that would come up if users tried it and add to the Troubleshooting Guide.

Reply:
Agree with @bruno here. I'd recommend moving it to be the second header (as an H2) and take it out of note format entirely.